Voidify (#44)
project-trident/trident-website: hugo-site/content/post/void-20.02-available.md

+++
title = "Void Linux 20.02 Image Available"
menutitle = "[20.02] Void Download Available"
description = "The 20.02 install image is now available"
date = "2020-02-14"
draft = false
pre = "<i class='fa fa-exclamation'></i>	"
+++

> Project Trident is pleased to announce the first official release image based on Void Linux, available on the [Project Trident download page](/download).

Reviewer: ```suggestion
Project Trident is pleased to announce that the first official release image based on Void Linux is now available on the [Project Trident download page](/download).
```

> Please note the four different "levels" of an installation that are available in the Project Trident installer:

Reviewer: ```suggestion
Please note the Project Trident installer supports four different installation "levels":
```

* **Void** : Only the base-system from Void Linux and ZFS-related bootloader packages are installed.
* **Server** : A CLI-based system with additional services and utilities installed from Project Trident (firewall, cron, autofs, wireguard, additional shells, etc.)
* **Lite Desktop** : Everything needed for a graphical desktop install using Lumina. No extra fluff.
* **Full Desktop** : The "Lite" install with quite a few additional end-user utilities (office suite, Telegram, multimedia apps, etc).

**Note:** These installation levels provide pre-defined lists of packages to install for user convenience. The installed system can be easily be changed afterwards using the built-in package system.

## I installed the ALPHA, should I reinstall for the Release?
Yes. There have been many changes to boot setup and dataset organization since the ALPHA image. These changes are difficult to make without a full reinstall.

> ## I installed the BETA, should I reinstall for the Release?

Reviewer: ```suggestion
## I installed the BETA, do I have to reinstall for the Release?
```

> No need. There it only takes a couple simple steps to migrate your BETA system to the RELEASE. These steps are listed in the [Migration Notes](https://github.com/project-trident/trident-docs/wiki/Converting-an-install-to-Project-Trident) on the Documentation wiki.

Reviewer: ```suggestion
No. There are a couple simple steps to migrate your BETA system to the RELEASE. These steps are listed in the [Migration Notes](https://github.com/project-trident/trident-docs/wiki/Converting-an-install-to-Project-Trident) on the Documentation wiki.
```
